src/middlewared/middlewared/client/client.py

```diff
@@ -110,6 +126,8 @@ def __init__(self, url, *args, **kwargs):
                             sock.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
                         except (AttributeError, socket.error):
                             pass
+                except ReservFDError:
+                    log.debug('Failed to reserv a privileged port')
```

Comment:
I think we should not handle this exception here, client should not be initialized if we failed to reserve port. This code will just crash client a few lines below in `WebSocket.__init__(self, sock, ...)` with `NameError: sock`

Reply:
Alert plugin already handles some exceptions in a special way in `__run_source` method, you can just add this new exception there.